examples/Bodypose-keypoints/sketch.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2018-2023 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+let video;
+let bodypose;
+let poses = [];
+
+function preload() {
+  //Load the handpose model.
+  bodypose = ml5.bodypose();
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // Create the video and hide it
+  video = createCapture(VIDEO);
+  video.size(width, height);
+  video.hide();
+
+  // Start detecting poses in the webcam video
+  bodypose.detectStart(video, gotPoses);
+}
+
+function draw() {
+  // Draw the webcam video
+  image(video, 0, 0, width, height);
+
+  // Draw all the tracked landmark points
+  for (let i = 0; i < poses.length; i++) {
+    let pose = poses[i];
+    for (let j = 0; j < pose.keypoints.length; j++) {
+      let keypoint = pose.keypoints[j];
+      // Only draw a circle if the keypoint's confidence is bigger than 0.2
+      if (keypoint.score > 0.2) {
```

Comment:
This is useful to show the possibility but I have a few questions about it:
* The example is simpler to look at without it, should we include it?
* If we include it, maybe a lower threshold is fine? (0.1?)
* Is `score` the term from the model itself? Another option is `confidence`. I don't have a preference I just seem to remember we're using `confidence` for the image classifiers (teachable machine, etc) so maybe we should be consistent?

Reply:
I included the check to make the example output look less confusing since MoveNet will try to fit all the landmark points inside the video even though a body part is clearly out of frame, but I would also be happy to remove it for simplicity. 
(The top image is without the confidence check, and the bottom image has the check).
<img width="200" alt="Screenshot 2023-09-01 141524" src="https://github.com/ml5js/ml5-next-gen/assets/93690311/dfadc9cf-0e34-42fc-97ac-9caf60d3f7e9">
<img width="200" alt="Screenshot 2023-09-01 141702" src="https://github.com/ml5js/ml5-next-gen/assets/93690311/c4908ae6-7698-4cb6-877b-6757dcf6aad9">

If we decide to include it I think 0.1 would be a good idea.

I believe `score` is the term used for tf.js landmark models. I do agree that it should be consistent across different features. Personally, I think `confidence` is a little more intuitive, but I also don't have a strong preference. 

Please let me know your thoughts!

Reply:
Ah, yes, in that case I think we should keep the if statement, it's confusing to see all the extra points that are out of frame!

I always hesitate to change the name of something from the original source model, but I think we could make the decision to use "confidence" across the board instead of "score". Any thoughts @MOQN @sproutleaf @QuinnHe @gohai? I'm fine either way!

Reply:
Just my personal opinion, "confidence" will make more sense to beginners and different naming conventions may easily create confusion. The "scores" are confidence scores in the end 😉.